scripts/webpack.config.js

```diff
@@ -1,66 +0,0 @@
-const path = require('path');
```

Comment:
Does the build pipeline definitely not use these files?

Reply:
Nope. Webpack was completely removed from daily-schedules. It's build folder structure didn't match the pipeline, which was the main reason to remove it.